pkgs/top-level/all-packages.nix

```diff
@@ -8530,8 +8532,6 @@ with pkgs;
 
   networkmanager-sstp = callPackage ../tools/networking/networkmanager/sstp { };
 
-  networkmanagerapplet = callPackage ../tools/networking/networkmanager/applet { };
```

Comment:
Maybe even `network-manager-applet`, as that is the upstream project name: https://gitlab.gnome.org/GNOME/network-manager-applet

Reply:
``networkmanager-applet`` follows the semantic of all the other networkmanager things.

Reply:
Well, other NetworkManager projects are VPN plug-ins, and they use different naming scheme upstream too: https://gitlab.gnome.org/GNOME/NetworkManager-openvpn

Reply:
If anything, we should have renamed the VPN plug-ins to use `NetworkManager` prefix (or `network-manager` if we want to avoid the caps). But really network-manager-applet is a separate project for controlling NetworkManager so how NetworkManager plug-ins are named should have no bearing on it.

Reply:
Feel free to rename it again but without any dashes or capital letters you can't read it.